src/clicache/src/CacheableDate.cpp

```diff
@@ -90,7 +90,7 @@ namespace Apache
           TimeSpan epochSpan = m_dateTime - EpochTime;
           int64_t millitime =
             epochSpan.Ticks / TimeSpan::TicksPerMillisecond;
-          m_hashcode = (int)millitime ^ (int)((int64)millitime >> 32);
+          m_hashcode = (int)millitime ^ (int)((int64_t)millitime >> 32);
```

Comment:
I'm not sure I follow why you think C++/CLI sources don't need to be converted.  Are you suggesting changing types like `int32` to be `System::Int32` instead?